client/src/ui/Form.jsx

```diff
@@ -326,6 +351,103 @@ export default class Form extends Component {
     }
   }
 
+  getGradeOptions() {
+      const gradeOptions = [];
+      this.gradeRecieved = ["A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", "F"];
+      this.gradeRecieved.forEach((grade) =>
+        gradeOptions.push({
+          "value": grade,
+          "label": grade
+        }))
+      
+      return gradeOptions
```

Comment:
I would suggest using `map` here
```suggestion
      return this.gradeReceived.map((grade) =>
        ({
          "value": grade,
          "label": grade
        }))
```